projects/batfish/src/org/batfish/grammar/cisco/CiscoLexer.g4

```diff
@@ -9435,6 +9435,11 @@ FORWARD_SLASH
    '/'
 ;
 
```

Comment:
This change will have a serious impact. I would be surprised if it did not break other things.